models/Redirect.php

```diff
@@ -569,6 +570,6 @@ public function delete()
      */
     public static function getStatusCodes(): array
     {
-        return self::$statusCodes;
+        return Config::getSystemConfiguration('redirects')['status_codes'] ?: self::$statusCodes;
```

Comment:
we should completely move the status codes list to the Symfony config(define the array list in the `default.yaml` of CoreBundle) instead of falling back to the hard coded property. So one can add to the list just by defining more codes in the config.yaml.

Also the node should be part of `pimcore` config not `pimcore_admin`.

These should be done:
- [ ] Add redirects node under pimcore config and define default status codes in default.yaml:
```yaml
pimcore:
    redirects:
        status_codes:
            300: 'Multiple Choices'
            301: 'Moved Permanently'
            302: 'Found'
            303: 'See Other'
            307: 'Temporary Redirect'

```
- [ ] Remove hardcoded values in Redirect.php